### PR TITLE
starter: start v2.0 etcd when WALUnknown

### LINKED ISSUE #13: unhandled etcd version
```
core@localhost ~ $ /usr/bin/etcd --data-dir=.
2015/03/24 17:12:35 starter: detected etcd version Unknown WAL in .
2015/03/24 17:12:35 starter: unhandled etcd version in .
panic: starter: unhandled etcd version in .

goroutine 16 [running]:
runtime.panic(0x709ce0, 0xc208001640)
	/usr/lib/go/src/pkg/runtime/panic.c:279 +0xf5
log.Panicf(0x8897b0, 0x25, 0xc2080dfb10, 0x1, 0x1)
	/usr/lib/go/src/pkg/log/log.go:314 +0xcd
github.com/coreos/etcd-starter/starter.checkInternalVersion(0xc208004480, 0x0, 0x0)
	/build/amd64-usr/var/tmp/portage/dev-db/etcd-starter-0.0.2/work/gopath/src/github.com/coreos/etcd-starter/starter/starter.go:168 +0xd26
github.com/coreos/etcd-starter/starter.StartDesiredVersion(0x87a0b0, 0x24, 0xc208001170, 0x1, 0x1)
	/build/amd64-usr/var/tmp/portage/dev-db/etcd-starter-0.0.2/work/gopath/src/github.com/coreos/etcd-starter/starter/starter.go:74 +0x260
main.main()
	/build/amd64-usr/var/tmp/portage/dev-db/etcd-starter-0.0.2/work/gopath/src/github.com/coreos/etcd-starter/main.go:76 +0x2de

goroutine 19 [finalizer wait]:
runtime.park(0x414820, 0xc59f98, 0xc46169)
	/usr/lib/go/src/pkg/runtime/proc.c:1369 +0x89
runtime.parkunlock(0xc59f98, 0xc46169)
	/usr/lib/go/src/pkg/runtime/proc.c:1385 +0x3b
runfinq()
	/usr/lib/go/src/pkg/runtime/mgc0.c:2644 +0xcf
runtime.goexit()
	/usr/lib/go/src/pkg/runtime/proc.c:1445

goroutine 24 [syscall]:
os/signal.loop()
	/usr/lib/go/src/pkg/os/signal/signal_unix.go:21 +0x1e
created by os/signal.init·1
	/usr/lib/go/src/pkg/os/signal/signal_unix.go:27 +0x32
```

### PR DESCRIPTION
**Pull request by yichengq**
fixes #13 
/cc @kelseyhightower @crawford

### CODE REVIEW

**Comment by barakmich** [comment]
While this is reasonable in etcd-starter, a more explicit "hey-this-directory-is-empty" response from etcd could be much better.

(LGTM)

**Comment by xiang90** [comment]
LGTM

File: starter/starter.go [hunk 1]
```diff
@@ -163,6 +163,9 @@ func checkInternalVersion(fs *flag.FlagSet) version {
 			return internalV2
 		}
 		return ver
+	case wal.WALUnknown:
+		log.Printf("starter: failed to understand data dir %s, and hand over to v2.0 etcd", dataDir)
```
Comment (crawford) [inline starter/starter.go]: "starter: unrecognized contents in data dir %s"

Comment (yichengq) [inline starter/starter.go]: done.

**Comment by yichengq** [comment]
@barakmich Sure.

File: starter/starter.go [hunk 2]
```diff
@@ -163,6 +163,9 @@ func checkInternalVersion(fs *flag.FlagSet) version {
 			return internalV2
 		}
 		return ver
+	case wal.WALUnknown:
+		log.Printf("starter: unrecognized contents in data dir %s, and hand over to v2.0 etcd", dataDir)
```
Comment (crawford) [inline starter/starter.go]:
```go
log.Printf("starter: unrecognized contents in data directory %s", dataDir)
```

**Comment by crawford** [comment]
LGTM

**Pull request merged**